cache-config/t3c-apply/config/config.go

```diff
@@ -681,8 +681,7 @@ func getOSSvcManagement() SvcManagement {
 		_svcManager = SystemD
 	} else if isCommandAvailable(Service) {
 		_svcManager = SystemV
-	}
```

Comment:
Since systemV relies on chkconfig it may be best to do something like this: 

```if _svcManager == SystemV && !isCommandAvailable(Chkconfig) {
		return Unknown
	}```